ios/Classes/FlutterDocumentView.m

```diff
@@ -90,13 +103,24 @@ - (void)onMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
             config = nil;
         }
         
-        [self openDocument:document password:password config:config];
+        [self openDocument:document password:password config:config resultToken:result];
+    } else if ([call.method isEqualToString:@"importAnnotationCommand"]) {
```

Comment:
Would be good to change all of these string keys to defined constants. This hasn't really been done yet, but it should be. You can see at the top of the file a handful defined like this:

`static NSString * const PTDisabledToolsKey = @"disabledTools";`

We should do that for all cases where we're using a particular string as s key.